src/core/reference/include/ngraph/runtime/reference/binary_convolution.hpp

```diff
@@ -130,7 +130,7 @@ void binary_convolution(const T_IN* in,
 
     const size_t filters_count = filters_shape[filter_out_ch_axis];
     const Shape filter_shape(++filters_shape.begin(), filters_shape.end());
-    const size_t filter_size = shape_size(filter_shape);
+    const size_t filter_size = shape_size(filter_shape) / 8;
```

Comment:
What if filter_size % 8 != 0?

Reply:
@wilson-seok

Reply:
@ilyachur I couldn't understand why filter_size %8 != 0 case is considered. I think filter_size doesn't matter %8 =0 or not. Is there any restrict?

Reply:
@wilson-seok I would like to understand what if `shape_size(filter_shape) % 8 != 0`? Will it work? In this case We will round off the result

Reply:
@ilyachur I added the case of shape_size(filter_shape) % 8 != 0. Thanks!